pages/index.vue

```diff
@@ -16,7 +16,7 @@ export default {
 				username: false,
 				room: false,
 			},
-			attemptedToJoin: false, // Flag for tracking form submission attempt
```

Comment:
removed this because button to join is disabled if username and room name validation fails